BedrockCommand.cpp

```diff
@@ -123,6 +123,13 @@ bool BedrockCommand::areHttpsRequestsComplete() const {
     return true;
 }
 
+ map<string,string> BedrockCommand::getCommandLogParams() {
```

Comment:
I am not seeing how this returns all the parameters of the command 😕

Reply:
It doesn't return all parameters. I can make it return all parameters though, but I chose to list only accountID and email directly from Auth.